### ISSUE TITLE
create_model_and_transforms might give wrong normalizations

### ISSUE THREAD

**Issue by mattdeitke** [mlfoundations/open_clip#204]
I'm running some experiments for a paper and noticed that all LAION-400M and LAION-2B pre-trained models, except the LAION-2B ViT-L/14 model gave incredibly bad performance, much worse than expected and the OpenAI counterparts.

To put some numbers in context (higher is better, ignore the metrics, not important - **new norm** is the corrected normalization):

![image](https://user-images.githubusercontent.com/28768645/200392088-d827897e-281b-45ba-83ff-e87fe7b15416.png)

Digging around, all models, except the LAION-2B ViT-L/14 model, return a preprocessing normalization of `Normalize(mean=(0.48145466, 0.4578275, 0.40821073), std=(0.26862954, 0.26130258, 0.27577711))` from `create_model_and_transforms`. For instance:
![image](https://user-images.githubusercontent.com/28768645/200392569-b2835b23-8690-4476-8a22-1c849f3afb01.png)

I changed the normalization for all the LAION-400M and LAION-2B models to be `Normalize(mean=(0.5, 0.5, 0.5), std=(0.5, 0.5, 0.5))` and got much better performance or all models, and about what I'd expect (very similar to that of the OpenAI models). Note that the pretrained OpenAI models don't have this issue, and they work with the correctly supplied preprocess method. The numbers with the `0.5` normalization were:
![image](https://user-images.githubusercontent.com/28768645/200393185-53d825b4-0fe7-4b00-9d73-f55c2c8f0fa2.png)

It seems like the `preprocess` for the LAION-400M and LAION-2B models are all using `Normalize(mean=(0.5, 0.5, 0.5), std=(0.5, 0.5, 0.5))`, which should be returned from `create_model_and_transforms`.

**Comment by rwightman** [comment]
@mattdeitke I'm not following this exactly, the L/14 LAION-2B is the only model that should be returning transformers with 0.5, 0.5, 0.5, are you seeing that for other models?

**Comment by mattdeitke** [comment]
It seems like the models might have accidentally trained with `Normalize(mean=(0.5, 0.5, 0.5), std=(0.5, 0.5, 0.5))`.

All models are returning `Normalize(mean=(0.48145466, 0.4578275, 0.40821073), std=(0.26862954, 0.26130258, 0.27577711))` as expected. But the performance is very bad with that normalization. When I change the normalization for all models to `Normalize(mean=(0.5, 0.5, 0.5), std=(0.5, 0.5, 0.5))`, everything works as expected, and the performance is much better and more similar to that of the CLIP models.

**Comment by rwightman** [comment]
@mattdeitke I can assure you that other models here were not trained with 0.5, that option was only added at the time of my last training of L/14 (it wasn't merged until after Romain trained the g/14 H/14), it was only possible to do with manual intervention prior to that. And we have full validation of these models on various zero-shot tasks that matches the expectation.

If somehow this works better for your task, perhaps there is some odd normalization impact in your case?
